gaia-list.js

```diff
@@ -60,7 +60,6 @@ proto.shadowStyleHack = function() {
 
 proto.attachedCallback = function() {
```

Comment:
I have a feeling we may be able to remove this function and the one below altogether.